Makefile

```diff
@@ -100,16 +100,16 @@ remove-agents:
 
 # Backups
 backup_core:
-	./helper.sh --execute postgres-oms-core-elixir -- pg_dump 'postgresql://postgres:$${PW_POSTGRES}@localhost/omscore_dev' --inserts > oms_core.sql.backup-$(shell date +%Y-%m-%dT%H:%M)
+	./helper.sh --execute postgres-core- -- pg_dump 'postgresql://postgres:$${PW_POSTGRES}@localhost/core' --inserts > core.sql.backup-$(shell date +%Y-%m-%dT%H:%M)
```

Comment:
in `postgres-core-`, there's an extra `-`

Reply:
Oops, fixed